tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala

```diff
@@ -455,4 +455,20 @@ class HoverTermSuite extends BaseHoverSuite {
       "3" -> "case CONTINUE: FileVisitResult".hover
     )
   )
+
+  check(
+    "toplevel".tag(IgnoreScala2),
+    """
+      |val (first, <<se@@cond>>) = (1, false)
+      |""".stripMargin,
+    """|```scala
+       |package 
+       |```
+       |""".stripMargin,
+    compat = Map(
+      "3.1.3" ->
```

Comment:
Maybe we should be able to express "3.1.3 or higher" by `compat` (of course we don't need to handle this PR)?
Looks like now we have to add `3.1.4`, `3.1.5`, ... patterns for every new scala3 release.

Reply:
Maybe we can use https://github.com/vdurmont/semver4j

Reply:
We already have some mechanisms for doing this as well as out own `SemVer`, so I just reused that.